modules/application/src/main/java/edu/northeastern/truthtree/adapter/stories/StoriesDBAdapter.java

```diff
@@ -94,42 +89,34 @@ public StoryDTO createStory(StoryDTO storyDTO) {
   }
 
   @Override
-  public List<StoryDTO> getApprovedStories() {
-    UriComponentsBuilder builder = UriComponentsBuilder.fromHttpUrl(STORIES_URL_GET_APPROVED);
-    String url = builder.toUriString();
-    String jsonResponse = URLUtil.readJSONFromURLInString(url);
-    return assembler.fromJSONStringToDTOList(jsonResponse);
-  }
-
-  @Override
-  public List<StoryDTO> getPendingStories() {
-    UriComponentsBuilder builder = UriComponentsBuilder.fromHttpUrl(STORIES_URL_GET_PENDING);
-    String url = builder.toUriString();
-    String jsonResponse = URLUtil.readJSONFromURLInString(url);
-    return assembler.fromJSONStringToDTOList(jsonResponse);
-  }
-
-  @Override
-  public StoryDTO approveStory(String id) {
+  public String changeStatusStory(String status, String id) {
```

Comment:
hmm, maybe just me but return `String` feel a bit awkward, we can prolly have a better way to handle success/error, but you dont have to worry abt it till we come up with a unified way to handle errors i guess.